docs/_includes/_replacements.rst

```diff
@@ -1,30 +1,106 @@
+.. |BertSequenceEncodingExtractor| replace:: :py:class:`BertSequenceEncodingExtractor`
+.. |BertLMExtractor| replace:: :py:class:`BertLMExtractor`
+.. |BertSentimentExtractor| replace:: :py:class:`BertSentimentExtractor`
+.. |ComplexTextExtractor| replace:: :py:class:`ComplexTextExtractor`
+.. |DictionaryExtractor| replace:: :py:class:`DictionaryExtractor`
+.. |NumUniqueWordsExtractor| replace:: :py:class:`NumUniqueWordsExtractor`
+.. |PartOfSpeechExtractor| replace:: :py:class:`PartOfSpeechExtractor`
+.. |SpaCyExtractor| replace:: :py:class:`SpaCyExtractor`
+.. |TextVectorizerExtractor| replace:: :py:class:`TextVectorizerExtractor`
+.. |VADERSentimentExtractor| replace:: :py:class:`VADERSentimentExtractor`
+.. |WordCounterExtractor| replace:: :py:class:`WordCounterExtractor`
+.. |WordEmbeddingExtractor| replace:: :py:class:`WordEmbeddingExtractor`
+.. |FarnebackOpticalFlowExtractor| replace:: :py:class:`FarnebackOpticalFlowExtractor`
+.. |TensorFlowKerasApplicationExtractor| replace:: :py:class:`TensorFlowKerasApplicationExtractor`
+.. |TFHubExtractor| replace:: :py:class:`TFHubExtractor`
+.. |TFHubImageExtractor| replace:: :py:class:`TFHubImageExtractor`
+.. |TFHubTextExtractor| replace:: :py:class:`TFHubTextExtractor`
+
 
 .. currentmodule:: pliers.converters
 .. |Converter| replace:: :py:class:`Converter`
+.. |ComplexTextIterator| replace:: :py:class:`ComplexTextIterator`
+.. |MicrosoftAPITextConverter| replace:: :py:class:`MicrosoftAPITextConverter`
 .. |AudioToTextConverter| replace:: :py:class:`AudioToTextConverter`
 .. |IBMSpeechAPIConverter| replace:: :py:class:`IBMSpeechAPIConverter`
 .. |VideoToAudioConverter| replace:: :py:class:`VideoToAudioConverter`
+.. |VideoToComplexTextConverter| replace:: :py:class:`VideoToComplexTextConverter`
+.. |VideoToTextConverter| replace:: :py:class:`VideoToTextConverter`
 .. |GoogleVisionAPITextConverter| replace:: :py:class:`GoogleVisionAPITextConverter`
 .. |TesseractConverter| replace:: :py:class:`TesseractConverter`
 .. |VideoFrameIterator| replace:: :py:class:`VideoFrameIterator`
+.. |VideoFrameCollectionIterator| replace:: :py:class:`VideoFrameCollectionIterator`
 .. |GoogleSpeechAPIConverter| replace:: :py:class:`GoogleSpeechAPIConverter`
+.. |RevAISpeechAPIConverter| replace:: :py:class:`RevAISpeechAPIConverter`
+.. |WitTranscriptionConverter| replace:: :py:class:`WitTranscriptionConverter`
+.. |ExtractorResultToSeriesConverter| replace:: :py:class:`ExtractorResultToSeriesConverter`
+
 
 .. currentmodule:: pliers.filters
 .. |Filter| replace:: :py:class:`Filter`
 .. |FrameSamplingFilter| replace:: :py:class:`FrameSamplingFilter`
 .. |ImageCroppingFilter| replace:: :py:class:`ImageCroppingFilter`
+.. |PillowImageFilter| replace:: :py:class:`PillowImageFilter`
+.. |PunctuationRemovalFilter| replace :py:class:`PunctuationRemovalFilter`
+.. |TokenizingFilter| replace :py:class:`TokenizingFilter`
+.. |TokenRemovalFilter| replace :py:class:`TokenRemovalFilter`
```

Comment:
Why is this happening? Anyone have any idea? Looks like a valid import?

@tyarkoni ?